tests/e2e.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { test, expect, defaultAddress, paymentMethod } from '../fixtures/fixture';
+
+test('E2E flow', async ({ loggedInPage, homePage, productPage, cartPage, checkoutPage }) => {
+    const page = loggedInPage.page;
+    await page.goto('/');
+
+    const productName = await homePage.getFirstProductName();
+    const productPrice = await homePage.getProductPrice();
+    
+    await homePage.clickFirstProduct();
+
+    await expect(productPage.title).toHaveText(productName);
+    await expect(productPage.price).toHaveText(productPrice);
+    await productPage.addCartButton.click();
+    await productPage.cartIcon.click();
+
+    await expect(page.getByTestId('product-title')).toHaveText(productName);
+    const actualPrice =  await cartPage.getActualPrice();
+    expect(Number(productPrice.replace('$', '').trim())).toBeCloseTo(actualPrice);
+
+    await checkoutPage.checkoutButton.click();
+    await expect(page.getByText(/you are already logged in/i)).toBeVisible();
+    await checkoutPage.checkoutButton.click();
+
+    await checkoutPage.ProceedCheckout(
```

Comment:
I'd use destructuring here

const {.street, city, .... } = defaultAddress;

await checkoutPage.ProceedCheckout(street, city, ...) {}

The same for FillPayment method. 

Important: please use camelCase to create variables and functions